.vib/sonarqube/goss/goss.yaml

```
@@ -0,0 +1,25 @@
+file:
+  /bitnami/sonarqube:
+    exists: true
+    filetype: directory
+    mode: "2775"
+    owner: root
+  /opt/bitnami/sonarqube/conf/sonar.properties:
+    exists: true
+    filetype: file
+    contains:
+     - sonar.web.port={{ .Vars.containerPorts.http }}
+     - sonar.search.port={{ .Vars.containerPorts.elastic }}
+     - sonarqube-postgresql:{{ .Vars.postgresql.service.ports.postgresql }}/{{ .Vars.postgresql.auth.database }}
+  /var/run/secrets/kubernetes.io/serviceaccount:
+    exists: {{ .Vars.serviceAccount.automountServiceAccountToken }}
+command:
+  check-user-info:
+    exec: id
+    exit-status: 0
+    stdout:
+      - uid={{ .Vars.containerSecurityContext.runAsUser }}
+      - /groups=.*{{ .Vars.podSecurityContext.fsGroup }}/
+http:
+  http://localhost:{{ .Vars.service.ports.elastic }}:
+    status: 200
```

This does not add any value if we are already checking the port in `sonar.properties`

---

Shouldn't this be checking the elastic port in the K8s svc like:

```suggestion
http:
  http://sonarqube:{{ .Vars.service.ports.elastic }}:
    status: 200
```

---

This is not working:

```
http://sonarqube:{{ .Vars.service.ports.elastic }}:
```
It only works with localhost. I can either remove this test, leave it as localhost, or it should be fixed in the chart.

---

In that case we'll need to create an internal task to work on this bug. I suggest removing this test and adding to the related task an AC where its inclusion is evaluated (if necessary).